scripts/processing_server.py

```diff
@@ -0,0 +1,98 @@
+import os
+import xeauth
+import pymongo
+import rframe
+import requests
+from fastapi import FastAPI, Request, Depends, status, HTTPException
+from fastapi.security import HTTPBearer
+from typing import Any, Optional, Union, List 
+
+from utilix.processing_requests import ProcessingRequest, ProcessingJob
+
+schemas = {
+    'processing_request': ProcessingRequest,
+    'processing_job': ProcessingJob,
+}
+
+mongo_user = os.getenv('MONGO_USER')
```

Comment:
Sure I can first check the utilix config, but this file is a demo api server, so it would only be running on one machine. Didn't want to add extra credentials to utilix config file that will only be used on one machine...

Reply:
Yeah it's fine as is of course, I just think it's nice to standardize how the client gets set up. Then can just have a specific config on that specific machine, if needed.

Reply:
done.